t/full-stack.t

```diff
@@ -523,5 +524,8 @@ END {
     kill_driver;
     turn_down_stack;
     session->clean;
+    # prevent double execution
+    return if $? == 0;
```

Comment:
I still think this is wrong. Somewhere there's an improperly done `fork` causing all sorts of random issues and this masks the most obvious symptom. This needs to be properly investigated.

Reply:
I removed that part now